modules/ipi-install-troubleshooting_proc_worker-nodes-cannot-join-the-cluster.adoc

```diff
@@ -0,0 +1,17 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="worker-nodes-cannot-join-the-cluster_{context}"]
+= Worker nodes cannot join the cluster
+
+Installer-provisioned clusters deploy with a DNS server that includes a DNS entry for the `api-int.<cluster_name>.<base_domain>` URL. If the nodes within the cluster are using an external or an upstream DNS server to resolve the `api-int.<cluster_name>.<base_domain>` URL and there is no such entry, worker nodes may fail to join the cluster.
+
+.Procedure
+
+. Add a DNS A/AAAA or CNAME record to internally identify the API load balancer.
+
+. Add a DNS PTR record to internally identify the API load balancer. 
+
+These records must be resolvable from all the nodes within the cluster.
```

Comment:
There are many different ways, because there are different DNS systems such as dnsmasq, DNS/BIND, and systems at domain registrars among others. I've modified the procedure to provide a dnsmasq example for each step.